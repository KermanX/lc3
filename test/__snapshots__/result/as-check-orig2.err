Error at line 3: Duplicate .ORIG directive