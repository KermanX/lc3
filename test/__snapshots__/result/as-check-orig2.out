ADD     R1, R2, R3             0001001010000011